packages/app/src/runner/SpecRunner.vue

```diff
@@ -311,7 +320,7 @@ $navbar-width: 80px;
 
 #unified-runner {
   position: relative;
-  margin: 0 auto;
+  margin: v-bind('runnerMargin');
```

Comment:
I feel like these will be *really* hard to debug. So far, we have manage to keep all the screenshot related logic in components named `<RemoveClassesDuringScreenshotting>`, etc. Can we do something similar here? The `<div :id="RUNNER_ID">` is already inside of `RemoveClassesDuringScreenshotting`, maybe we can make the name more generic, like `ModifyStyleDuringScreenshotting`.

My previous experience with the screenshotting logic getting spread all throughout the app is it gets really messy - so if possible, it'd be great to keep it in one place, so if we have a problem, we know exactly where to look.